synthetic_data/generator_builder.py

```diff
@@ -22,14 +22,15 @@ class Generator:
 
     @classmethod
     def is_valid_data(cls, profile):
+        """Check if profile's data is valid."""
         return profile.__class__ in cls.valid_data_types
 
     def __new__(cls, seed=None, config=None, *args, **kwargs):
         """Instantiate proper generator."""
         if config:
             try:
                 return config(*args, **kwargs)
-            except Exception as e:
+            except Exception as e:  # noqa F841
```

Comment:
should be careful when doing `noqa` .... F841 is `Local variable name is assigned but not used` ... I think what should happen here is that `as e` is removed